ddtrace/commands/ddtrace_run.py

```diff
@@ -58,9 +56,15 @@ def main():
     )
     parser.add_argument("command", nargs=argparse.REMAINDER, type=str, help="Command string to execute.")
     parser.add_argument("-d", "--debug", help="enable debug mode (disabled by default)", action="store_true")
-    parser.add_argument("-i", "--info", help="print library info useful for debugging", action="store_true")
+    parser.add_argument(
+        "-i",
+        "--info",
+        help="print library info useful for debugging. Does not reflect configuration changes made during runtime",
```

Comment:
```suggestion
        help="print library info useful for debugging. Does not reflect configuration changes made during runtime",
```

Not sure the best wording but I'd like to be a bit more direct that this is really only useful when configuring `ddtrace` with environment variables.

"Does not reflect configuration changes made during runtime" does capture this but other people's interpretations of runtime may vary.

Reply:
went with this: _Only reflects configurations made via environment variables, not those made in code._ 
But let me know if you have something different to suggest.